WordPress/Classes/Services/MediaLibrary.swift

```diff
@@ -132,6 +132,7 @@ open class MediaLibrary: LocalCoreDataService {
         var options = MediaImageExporter.Options()
         options.maximumImageSize = self.exporterMaximumImageSize()
         options.stripsGeoLocationIfNeeded = MediaSettings().removeLocationSetting
+        options.imageCompressionQuality = 0.9
```

Comment:
Good idea. I've addd this within the context of `MediaLibrary` itself, and also where it should be used between `MediaLibrary` and `MediaLibraryPickerDataSource`. Other media code using the `0.9` value will be wiped out soon, when we switch to using the exporters.